canonical-web-and-design/ubuntu.com
# Added /data redirect

## Done

- Added temporary redirect from `/data` to `/blog/tag/data-lake`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- See that you have properly been redirected 


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10519
